gto-support-scarlet/src/main/kotlin/com/tinder/scarlet/internal/utils/RuntimePlatformInternals.kt

```diff
@@ -0,0 +1,3 @@
+package com.tinder.scarlet.internal.utils
+
+internal fun defaultPlatform() = RuntimePlatformInternals.Default()
```

Comment:
we wrap the package-private java `RuntimePlatformInternals.Default()` in an internal Kotlin function, this allows us to access it from other packages in this module without exposing it on the public API